app/frontend/actions/common.ts

```diff
@@ -0,0 +1,108 @@
+import {State, Store} from '../state'
+import {getWallet} from './wallet'
+import {
+  AssetFamily,
+  DelegateTransactionSummary,
+  DeregisterStakingKeyTransactionSummary,
+  Lovelace,
+  SendTransactionSummary,
+  TxPlanArgs,
+  TxType,
+  WithdrawTransactionSummary,
+} from '../types'
+import {TxPlan, TxPlanResult} from '../wallet/shelley/shelley-transaction-planner'
+
+export default (store: Store) => {
+  const {setState, getState} = store
+
+  const resetTransactionSummary = (state: State) => {
+    setState({
+      sendTransactionSummary: {
+        // TODO: we should reset this to null
+        type: TxType.SEND_ADA,
+        address: null,
+        coins: 0 as Lovelace,
+        token: null,
+        minimalLovelaceAmount: 0 as Lovelace,
+        fee: 0 as Lovelace,
+        plan: null,
+      },
+    })
+  }
+
+  const resetSendFormState = (state: State) => {
```

Comment:
If I see correctly, this is only used in `submitTransaction` so it could just be part of `action/transactions` and would not have to be exported.

Reply:
Fixed